inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/EncryptConfigEntry.java

```diff
@@ -99,20 +108,20 @@ public EncryptInfo getRsaEncryptInfo() {
         synchronized (this.lastUpdateTime) {
             if (visitTime == this.lastUpdateTime.get()) {
                 RSAPublicKey rsaKey = EncryptUtil.loadPublicKeyByText(pubKey);
-                this.desKey = EncryptUtil.generateDesKey();
+                this.aesKey = EncryptUtil.generateAesKey();
                 try {
-                    byte[] encryptedKey = EncryptUtil.rsaEncrypt(rsaKey, this.desKey);
+                    byte[] encryptedKey = EncryptUtil.rsaEncrypt(rsaKey, this.aesKey);
                     String tmpKey = Base64.encodeBase64String(encryptedKey);
                     rsaEncryptedKey = URLEncoder.encode(tmpKey, "utf8");
                     this.lastUpdateTime.set(System.currentTimeMillis());
-                    return new EncryptInfo(this.version, this.rsaEncryptedKey, this.desKey);
+                    return new EncryptInfo(this.version, this.rsaEncryptedKey, this.aesKey);
```

Comment:
These codes is same with getRsaEncryptedKey.

Reply:
Looking into that in future PRs.

Reply:
Looking into the code structure problem in separate threads.